contents/docs/self-host/deploy/aws.mdx

```diff
@@ -82,12 +81,6 @@ along to upstream PostHog services.
 
 <SetupDNS />
 
-## Troubleshooting
```

Comment:
Removed as this is not relevant anymore. In any case it should belong to the troubleshooting section, not installation